js/common/bootstrap/errors.js

```diff
@@ -340,6 +340,7 @@
     "ERROR_SUPERVISION_GENERAL_FAILURE" : { "code" : 20501, "message" : "general supervision failure" },
     "ERROR_QUEUE_FULL"             : { "code" : 21003, "message" : "queue is full" },
     "ERROR_QUEUE_TIME_REQUIREMENT_VIOLATED" : { "code" : 21004, "message" : "queue time violated" },
+    "ERROR_TOO_MANY_DETACHED_THREADS" : { "code" : 21005, "message" : "too many detached scheduler threads" },
```

Comment:
note: this change does not really belong to this PR, but a previous PR changed `lib/Basics/errors.dat` without adding the changed generated JavaScript file `errors.js`. 
My PR now does that, because otherwise one will always end up with a modified local file after building 3.11.